js/poloniex.js

```diff
@@ -1778,6 +1780,114 @@ module.exports = class poloniex extends Exchange {
+        //                 "childChains": []
+        //             }
+        //         }
+        //     ]
+        //
+        // console.log (response);
+        return this.parseDepositWithdrawFees (response, codes);
+    }
+
+    parseDepositWithdrawFees (response, codes = undefined, currencyIdKey = undefined) {
+        //
+        //     [
+        //         {
+        //             "1CR": {
+        //                 "id": 1,
+        //                 "name": "1CRedit",
+        //                 "description": "BTC Clone",
+        //                 "type": "address",
+        //                 "withdrawalFee": "0.01000000",
+        //                 "minConf": 10000,
+        //                 "depositAddress": null,
+        //                 "blockchain": "1CR",
+        //                 "delisted": false,
+        //                 "tradingState": "NORMAL",
+        //                 "walletState": "DISABLED",
+        //                 "parentChain": null,
+        //                 "isMultiChain": false,
+        //                 "isChildChain": false,
+        //                 "childChains": []
+        //             }
+        //         }
+        //     ]
+        //
+        const depositWithdrawFees = {};
+        const childChainValues = {};
+        codes = this.marketCodes (codes);
+        for (let i = 0; i < response.length; i++) {
+            const entry = response[i];
+            const currencyId = this.safeString (Object.keys (entry), 0);
+            const code = this.safeCurrencyCode (currencyId);
+            const feeInfo = entry[currencyId];
+            if ((codes === undefined) || (this.inArray (code, codes))) {
+                depositWithdrawFees[code] = this.depositWithdrawFee ({});
+                depositWithdrawFees[code]['info'][currencyId] = feeInfo;
+                const networkId = this.safeString (feeInfo, 'blockchain');
+                const withdrawFee = this.safeNumber (feeInfo, 'withdrawalFee:');
+                const withdrawResult = {
+                    'fee': withdrawFee,
+                    'percentage': (withdrawFee !== undefined) ? false : undefined,
+                };
+                const depositResult = {
+                    'fee': undefined,
+                    'percentage': undefined,
+                };
+                const networkCode = this.networkIdToCode (networkId);
+                depositWithdrawFees[code]['networks'][networkCode] = {
+                    'withdraw': withdrawResult,
+                    'deposit': depositResult,
+                };
+                const childChains = this.safeValue (feeInfo, 'childChains');
+                if (childChains.length > 0) {
+                    for (let j = 0; j < childChains.length; j++) {
+                        childChainValues[childChains[j]] = code;
+                    }
+                }
+            }
+        }
+        // TODO: Include sideChains
```

Comment:
@AlgoQ is this PR ready?

Reply:
Oops it seems like I made a git mistake, I made a branch from my [WIP - poloniex PR](https://github.com/ccxt/ccxt/pull/16532) which I've updated later. I'm going to remove the poloniex file from this PR